portfolio/src/main/webapp/index.html

```diff
@@ -5,48 +5,38 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
+    <script src="script2.js"></script>
   <body>
     <div id="content">
       <h1>Cynthia's Website</h1>
       <p>Welcome to My portfolio.</p>
       <p>My name is Cynthia this is my basic website I like boba</p>
-      <img src="/images/IMG_9685.HEIC" />
+      <img src="images/image1.jpg">
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+      <div id="greeting-container">
 
-   <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>hey</title>
-  </head>
-  <body>
-    <h1>heyyyy</h1>
-    <p>Click <a href="/hey">here</a> to view content generated by a servlet.</p>
-  </body>
-</html>
+       <div id="content">
 
-    <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
+            <title>hey</title>
+        
+            <h1>heyyyy</h1>
+            <p>Click <a href="/hey">here</a> to view content generated by a servlet.</p>
+ 
     <title>Fetch</title>
     <link rel="stylesheet" href="style.css">
```

Comment:
Stylesheets should only be in the head section, and I believe this one is already included there. The title is also set there so you should remove it from here.